DataFormats/L1TrackTrigger/interface/TTTrack.h

```diff
@@ -31,21 +31,24 @@ class TTTrack : public TTTrack_TrackWord {
   double theRInv;
   double thePhi;
   double theTanL;
-  double theEta;
   double theD0;
   double theZ0;
   unsigned int thePhiSector;
+  unsigned int theEtaSector;
   double theStubPtConsistency;
   double theChi2;
-  unsigned int NumFitPars;
+  double theChi2XY;
+  double theChi2Z;
+  unsigned int numFitPars;
   unsigned int theHitPattern;
-  float an_MVA_Value;
-  float another_MVA_Value;
-  int theTrackSeed;
+  double theTrkMVA1;
+  double theTrkMVA2;
+  double theTrkMVA3;
+  int theTrackSeedType;
   double theBField;  // needed for unpacking
   static constexpr unsigned int Npars4 = 4;
   static constexpr unsigned int Npars5 = 5;
-  static constexpr float MagConstant = 0.3;
+  static constexpr float MagConstant = 0.299792458;
```

Comment:
better to define this in terms of `CLHEP::c_light`?

Reply:
done